.vendor/mcp-entrypoint/MCPEntrypoint.ts

```diff
@@ -59,7 +59,7 @@ export abstract class DurableMCP<
       return await object.onSSE(c.req.raw) as unknown as Response
     })
 
-    router.post(path + '/message', cors(corsOptions), async (c) => {
+    router.post('/sse/message', cors(corsOptions), async (c) => {
```

Comment:
We want to be able to use the mounted prefix here, but we only have `path` within `mount` but this endpoint is defined on line 28 where we initialize the transport:

```ts
    this.transport = new SSEEdgeTransport(
      '/sse/message',
      this.ctx.id.toString()
    )
```

I wasn't sure how to best square this circle so I left it hard-coded